pkg/values/pb/values.pb.go

```diff
@@ -2,7 +2,7 @@
 // versions:
 // 	protoc-gen-go v1.31.0
 // 	protoc        v4.25.1
-// source: values.proto
```

Comment:
@ettec Can you remove changes to this file? It's not relevant to the change you're making here